lib/pbench/test/unit/agent/test_tool_data_sink.py

```diff
@@ -26,10 +25,9 @@ class TestBenchmarkRunDir:
     """
 
     @pytest.fixture
-    def cleanup_tmp(self, pytestconfig):
-        TMP = Path(pytestconfig.cache.get("TMP", None))
-        self.int_pb_run = TMP / "pbench-run-int"
-        self.ext_pb_run = TMP / "pbench-run-ext"
+    def cleanup_tmp(self, tmp_path):
+        self.int_pb_run = tmp_path / "pbench-run-int"
+        self.ext_pb_run = tmp_path / "pbench-run-ext"
         yield
         try:
             shutil.rmtree(self.int_pb_run)
```

Comment:
If `rmtree()` fails, here, do we get the behavior that we desire?  (I.e., I think line 35 will probably result in any exception being silently ignored.)  Ditto for line 39.

(Since we expect the removals to succeed, and since we take the same actions for each failure, we might as well do them in the same `try` block, no?)

Reply:
Not sure, but I think the goal was to squash the exception. Would have to look at the history of this test to be sure.

Reply:
History doesn't help.  This code landed with the containerized Tool Meister work in PR #2076, see https://github.com/distributed-system-analysis/pbench/pull/2076/files#diff-a43ec41cd50cf299ee7d47c0bc84490551777c991833971e49984fe54cf5b0c5R35.

Reply:
Fixed.